ReactiveCocoa/Swift/Property.swift

```diff
@@ -490,29 +493,18 @@ public final class Property<Value>: PropertyProtocol {
 		// they see a consistent view of the `self.value`.
 		// https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3042
 		let producer = unsafeProducer.replayLazily(upTo: 1)
-		
+
+		let atomic = Atomic<Value?>(nil)
+		disposable = producer.startWithNext { atomic.value = $0 }
```

Comment:
Using `Disposable` should mean less hops, and we have just this one client listening for the deinitialization anyway.